test/unit/ngsiv2/MQTT_receive_measures-test2.js

```diff
@@ -403,4 +403,32 @@ describe('MQTT: Measure reception ', function () {
             });
         });
     });
+    describe('When a POST single Raw measure arrives for the HTTP binding', function () {
+        beforeEach(function () {
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartgondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post(
+                    '/v2/entities?options=upsert',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/singleMeasuresRawTypes1.json')
+                )
+                .reply(204);
+        });
+        it('should send its value to the Context Broker', function (done) {
+            mqttClient.publish('/1234/MQTT_2/attrs/humidity', '0x4D', null, function (error) {
```

Comment:
```suggestion
            mqttClient.publish('/1234/MQTT_2/attrs/humidity', 'A$Ci1', null, function (error) {
```

Even if the tests works propperly, I suggest to change the message to send (`0x4D`) to another text since it is confusing. You are sending An ascii representation of an hex value. It is clear to put a normal ascii string. In the previous suggestion, the message is `A$Ci1`. The expexted result is `4124436931`